frontend/src/components/teacher/view-sessions/TestSessionListItem.tsx

```diff
@@ -39,6 +40,16 @@ const STATUS_LABELS = {
   upcoming: "Scheduled",
   past: "Assigned",
 };
+const STATUS_BACKGROUND_COLORS = {
+  active: "green.50",
+  upcoming: "yellow.50",
+  past: "grey.100",
+};
+const STATUS_COLORS = {
+  active: "green.400",
+  upcoming: "yellow.300",
+  past: "grey.400",
+};
```

Comment:
is this used anywhere else? if so can we move to const file?

Reply:
Right now it's only used in this file. I think that since we can reuse the list item component in all the places where that component is needed, we probably won't ever need these colours elsewhere. Did you have a specific case where you thought they might be needed?